.github/workflows/build-and-push-docker-image.yml

```diff
@@ -0,0 +1,33 @@
+---
+
+name: 🏗️
+
+on:   # yamllint disable-line rule:truthy
+  pull_request:
+  push:
+    branches: ["release/*", "unstable/*"]
+    tags: ["*"]
```

Comment:
When we push the tag first, there's going to be some time when the new version is out already, but the container doesn't actually exist yet. Therefore, I prefer building my release automation processes around the `workflow_dispatch` trigger, so that I'm able to type in the target version in the UI, get the image published and only then push a tag, only when it's known that the image exists already. Let's try reworking this with similar philosophy in mind.

Reply:
Thanks for clarifying. I've replaced the tags trigger with `workflow_dispatch` instead (049447a).